caching-service/build.gradle

```diff
@@ -54,8 +54,6 @@ dependencies {
     api project(':apiml-tomcat-common')
     api project(':onboarding-enabler-spring')
 
-
-    implementation libs.zowe.attls
```

Comment:
Why caching service is not using AT-TLS?

Reply:
it's transitive from apiml-tomcat-common